test/examples/embeddables/adding_children.ts

```diff
@@ -33,6 +33,11 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
     it('Can create a new child', async () => {
       await testSubjects.click('embeddablePanelToggleMenuIcon');
       await testSubjects.click('embeddablePanelAction-ACTION_ADD_PANEL');
+
+      // this seem like an overkill, but clicking this button which opens context menu was flaky
+      await testSubjects.waitForDeleted('savedObjectFinderLoadingIndicator');
+      await testSubjects.waitForEnabled('createNew');
+
       await testSubjects.click('createNew');
```

Comment:
Another option would be to click the `createNew` button and then verify that the popup is open, and if doesn't open just retry clicking the `createNew` button.